extract_endpoint/mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy]
+strict_optional = True
+ignore_missing_imports = True
+warn_unused_ignores = True
```

Comment:
github seems put off by your lack of a newline at the end of this file.

Reply:
line added to appease the github gods, thanks!